dashboard/context_processors.py

```diff
@@ -32,4 +32,4 @@ def course_view_option(request):
     return {"course_view_option": course_view_option}
 
 def last_updated(request):
-    return {'last_updated' : get_last_cron_run()}
\ No newline at end of file
+    return {'last_updated' : get_canvas_data_date()}
```

Comment:
should we compare the last update time from both UDW and CRON, and show the earliest of the two?

Reply:
Hmm, I'd believe that this UDW time should always be the right one as the table is refreshed when we update? 

The only way I think it might not be correct is if it's not present at all which is why I didn't completely remove the other method yet.

Reply:
Yes, the UDW table unizin_metadata is the one stores the latest update timestamp.

I will vote yes for the change and merge it in.